lib/util/cache/package/redis.ts

```diff
@@ -94,16 +97,28 @@ export async function init(
   }
   rprefix = prefix ?? '';
   logger.debug('Redis cache init');
-  client = createClient({
-    url,
+
+  const rewrittenUrl = url.replace(/^(redis|rediss)\+cluster:\/\//, '$1://');
```

Comment:
```suggestion
  const rewrittenUrl = url.replace(regEx(/^rediss?\+cluster:\/\//), '$1://');
```

Reply:
Updated to add unit tests around the application, used the `regEx` internal type, and simplified the RegEx.